extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -491,4 +496,19 @@ private boolean isEqual(State state1, State state2) {
         return colorModeIsEqual && effectIsEqual;
     }
 
+    @Override
+    public Collection<ConfigStatusMessage> getConfigStatus() {
+        // The bridge IP address to be used for checks
+        final String bridgeIpAddress = (String) getThing().getConfiguration().get(HOST);
+
+        // Check whether an IP address is provided
+        if (bridgeIpAddress == null) {
+
+            return ImmutableList.of(ConfigStatusMessage.Builder.error(HOST)
+                    .withMessageKeySuffix(HueConfigStatusMessage.IP_ADDRESS_MISSING.getMessageKey()).withArguments(HOST)
+                    .build());
+        }
```

Comment:
> @kaikreuzer what is your opinion regarding this?

I fully agree. The network address could be IPv6 or a hostname as well - both is valid.